src/bench.js

```diff
@@ -0,0 +1,46 @@
+import benchmark from "benchmark";
+import { AwsCrc32 } from "@aws-crypto/crc32";
+import { AwsCrtCrc32 } from "./AwsCrtCrc32.js";
+import { equal } from "assert";
+
+const generateBuffer = (size) => {
+  const buf = Buffer.alloc(size);
+  for (let i = 0; i < size; i++) buf[i] = parseInt(Math.random() * 256);
+  return buf;
+};
+
+const awsCrc32 = new AwsCrc32();
+const awsCrtCrc32 = new AwsCrtCrc32();
+
+for (const bufferSizeInKB of [16, 64, 256, 1024]) {
+  const suite = new benchmark.Suite();
+  const testBuffer = generateBuffer(bufferSizeInKB * 1024);
+
+  awsCrc32.update(testBuffer);
+  awsCrtCrc32.update(testBuffer);
```

Comment:
any change in results if buffer is updated in chunks of 16kb instead of all at once?

Reply:
To be taken up in follow-up PR.